### PR TITLE
mycli: depend on cffi

### PR DESCRIPTION
**Pull request by singingwolfboy**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mycli.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Mycli < Formula
   end
 
   depends_on "rust" => :build
+  depends_on "cffi"
```
Comment (nandahkrishna) [inline Formula/mycli.rb:23]: Shouldn't we also add `depends_on "pycparser"`?

Comment (chenrui333) [reply Formula/mycli.rb:23]: yeah, I think so

Comment (chenrui333) [reply Formula/mycli.rb:23]: the formula does not depend on `pycparser`?

Comment (nandahkrishna) [reply Formula/mycli.rb:23]: Sorry if I don't understand -- the `cffi` and `pycparser` resources were removed and only `depends_on "cffi"` was added. It doesn't seem like the formula already depends on `pycparser`. Should it be added or am I missing something here?

Comment (nandahkrishna) [reply Formula/mycli.rb:23]: Ah okay, just realised `cffi` depends on `pycparser` already.

**Review by nandahkrishna — APPROVED** [review]

File: Formula/mycli.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/mycli.rb:23]: I think the setup.py for mycli does not specify `pycparser` is actually the thing in here.

**Pull request merged**